modules/troubleshooting-cluster-upgrader.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-roxctl.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="troubleshooting_upgrader_{context}"]
+= Troubleshooting Cluster Upgrader
+
+[role="_abstract"]
+This section describes potential problems and their solutions when using legacy installation method for the secured cluster and enabling the automated updates.
+The errors described here can be found in the clusters view when the upgrader fails.
+
```

Comment:
add this above every heading per [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files):

[id="headingname_{context}"]

For example:

[id="upgrader-missing-permissions_{context}"]

Reply:
Done in 903310e71e47a36039766c05c92823e256187862 (will be squashed)